tools/arriba.wdl

```diff
@@ -0,0 +1,325 @@
+            ]
+        }
+        mark_duplicates: "Mark duplicates in the input BAM file with Arriba."
+        report_additional_columns_discard: "Report additional columns ['fusion_transcript', 'peptide_sequence', 'read_identifiers'] in the discarded fusions file."
+        fill_gaps: "Fill gaps in assembled transcripts with reference bases. Expands the fusion sequence to the complete sequence of the fusion gene."
+        max_e_value: "Maximum E-value for read support."
+        max_mismappers: "Maximum fraction of mismapped reads in the fusion region."
+        max_homolog_identity: "Maximum fraction of homologous sequence for genes."
+        max_kmer_content: "Maximum fraction of repetitive 3-mer content in the fusion region."
+        max_mismatch_pvalue: "Maximum p-value for mismatches in the fusion region."
+        quantile: "Genes with expression above the given quantile are eligible for filtering."
+        exonic_fraction: "Minimum fraction of exonic sequence between breakpoints."
+        coverage_fraction: "Minimum fraction of viral contig transcription."
+        min_itd_allele_fraction: "Minimum supporting read fraction for internal tandem duplications."
+        max_genomic_breakpoint_distance: "With 'wgs_svs', threshold for relating genomic and transcriptomic events."
+        min_supporting_reads: "Minimum number of supporting reads for a fusion."
+        homopolymer_length: "Maximum homopolymer length adjacent to breakpoints."
+        read_through_distance: "Minimum distance between breakpoints for read-through events."
+        min_anchor_length: "Minimum anchor length for split reads."
+        many_spliced_events: "Recover fusions with at least this many spliced breakpoints."
+        fragment_length: "For single-end data, this is the fragment length. With paired-end reads, this is ignored and determined automatically."
+        max_reads: "Subsample fusions with more than this number of reads."
+        top_n: "Only report the top N most highly expressed viral integration sites."
+        max_itd_length: "Maximum length of internal tandem duplications."
+        min_itd_supporting_reads: "Minimum number of supporting reads for internal tandem duplications."
+        modify_memory_gb: "Add to or subtract from dynamic memory allocation. Default memory is determined by the size of the inputs. Specified in GB."
+        modify_disk_size_gb: "Add to or subtract from dynamic disk space allocation. Default disk size is determined by the size of the inputs. Specified in GB."
+    }
+
+    input {
+        File bam
+        File gtf
+        File reference_fasta_gz
+        File? chimeric_sam
+        File? exclude_list
+        File? known_fusions
+        File? annotate_fusions
+        File? protein_domains
+        File? wgs_svs
+        Array[String] interesting_contigs = ["1", "2", "3", "4", "5", "6", "7", "8", "9", "10", "11", "12", "13", "14", "15", "16", "17", "18", "19", "20", "21", "22", "X", "Y", "AC_*", "NC_*"]
```

Comment:
I would expect all these to have `chr` prefixes. Does Arriba handle both prefix+non-prefix gracefully?

Reply:
Yes, fortunately Arriba handles both.

```
-i CONTIGS : Comma-/space-separated list of interesting contigs. Fusions between genes on other contigs are ignored. Contigs can be specified with or without the prefix chr. Asterisks (*) are treated as wild-cards. Default: 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18 19 20 21 22 X Y AC_* NC_*
```